src/services/doppler-legacy-client.doubles.ts

```diff
@@ -32,6 +32,7 @@ export class HardcodedDopplerLegacyClient implements DopplerLegacyClient {
     console.log(this.registerUser, model);
     await timeout(1500);
     return { success: true };
+    //return { expectedError: { registerDenied: true } };
```

Comment:
@lucianaDilizio comment?

Reply:
This is to test different responses in localhost.

Reply:
These comments are here to test the scenario easily in the local environment.

Reply:
Well I see both of you have the same idea about this. It should be nice in the future perhaps adding a comment stating the use of this line.

Reply:
Nah, if we all already know, the comment is not necessary :P